zerver/openapi/zulip.yaml

```diff
@@ -3884,6 +3884,36 @@ paths:
                                     ],
                                   "id": 1,
                                 }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to administrators when a user changes their [Let administrators export my private data]
+                                (/help/export-your-organization#configure-whether-administrators-can-export-your-private-data)
```

Comment:
Does this line-wrapping of a link work? Generally we should use a reference link in any case to keep the prose nice, and line-wrap a bit earlier.

Reply:
Fixed this and made some other revisions in the first commit, which I'll merge via a separate PR to lock in the API feature level change.